app/Auth/Access/RegistrationService.php

```diff
@@ -72,6 +73,13 @@ public function findOrRegister(string $name, string $email, string $externalId):
             ];
 
             $user = $this->registerUser($userData, null, false);
+        } else if ($updateUser && ($user->name !== $name || $user->email !== $email)) {
+            $user->update([
+                'name'  => $name,
+                'email' => $email,
+            ]);
+
+            Activity::add(ActivityType::USER_UPDATE, $user);
```

Comment:
Please could this be removed?
Activities generally reflect user actions or high-level processes, rather than raw changes to data. 
While the user data is technically being updated, no one is (intentionally) actioning a user update themselves.